ssv/spectest/tests/runner/preconsensus/post_decided.go

```diff
@@ -21,7 +21,10 @@ func PostDecided() tests.SpecTest {
 	decideRunner := func(r ssv.Runner, duty *types.Duty, decidedValue *types.ConsensusData, preMsgs []*types.SignedPartialSignatureMessage) ssv.Runner {
 		r.GetBaseRunner().State = ssv.NewRunnerState(3, duty)
 		for _, msg := range preMsgs {
-			r.ProcessPreConsensus(msg)
+			err := r.ProcessPreConsensus(msg)
+			if err != nil {
+				panic(err.Error())
```

Comment:
should we panic or handle the error?
Do we panic in other places?

Reply:
I decided to use `panic` because this is just for test generation. If there's an error, the generation will fail. Also, `panic` will facilitate to see the error and where it happened (since we weren't catching it before).